src/ZoneServer/Buffs/Handlers/Swordsman/Doppelsoeldner/DoublePayEarn.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Melia.Shared.Game.Const;
+using Melia.Zone.Buffs.Base;
+
+namespace Melia.Zone.Buffs.Handlers.Swordsman.Doppelsoeldner
+{
+	/// <summary>
+	/// Handle for the Bear Buff, which increases the target's defense.
```

Comment:
Double Pay Earn's and Shock's summary appear incorrect.

Reply:
This has been fixed.